.github/workflows/build.yaml

```diff
@@ -103,12 +103,21 @@ jobs:
           zstdcat build.tar.zst | tar x
           cd build
           cpack
+      - name: Install dev package
+        run: |
+          apt update
```

Comment:
Should be `update && upgrade` here? I'm not sure how problematic it will be on Ubuntu to have "mixed age" packages installed: the base image is cached and could have many month old packages installed alongside new packages installed here.